pkgs/applications/misc/stylish/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
```

Comment:
If your code does not use a C/C++ compiler, just use `stdenvNoCC`.

Reply:
Done